app/components/ch/components/BackToTopBtn.vue

```diff
@@ -1,5 +1,5 @@
 <template>
-  <button type="button" :class="classes">
+  <button type="button" :class="classes" aria-label="back-to-top-btn">
```

Comment:
@sylvainnicolet an aria label is a text read by a screen reader, it fullfills a missing descriptive text. You have to write them  in a natural language, not put an id here. Please replace `back-to-top-btn` with `back to top button`